www-apps/radicale/radicale-2.1.11-r2.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python3_6 python3_7 )
+
+inherit distutils-r1 eutils user systemd
+
+MY_PN="Radicale"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="A simple CalDAV calendar server"
+HOMEPAGE="https://radicale.org/"
+SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="+bcrypt"
+
+RDEPEND="sys-apps/util-linux
+	dev-python/vobject[${PYTHON_USEDEP}]
+	dev-python/python-dateutil[${PYTHON_USEDEP}]
+	bcrypt? ( dev-python/passlib[bcrypt,${PYTHON_USEDEP}] )"
+
+S=${WORKDIR}/${MY_P}
```

Comment:
add quotes

Reply:
563ebcb0cc1f4207ae53f70ac054ed73e687bbcf; I thought they weren’t strictly necessary for variable expansions inside other variable assignments, but I prefer being explicit too.